Source/SVWidgetsLib/FilterParameterWidgets/FilterParameterWidget.h

```diff
@@ -149,22 +149,6 @@ class SVWidgetsLib_EXPORT FilterParameterWidget : public QFrame
      */
     QPoint adjustedMenuPosition(QToolButton *pushButton);
 
-    /**
-     * @brief hasValidFilePath
-     * @param filePath
-     * @return
-     */
-    bool hasValidFilePath(const QString &filePath);
-
-    /**
-     * @brief verifyPathExists
-     * @param filePath
-     * @param lineEdit
-     * @return
-     */
-    bool verifyPathExists(QString filePath, QLineEdit* lineEdit);
```

Comment:
Have we tested this with all of our plugins?  Removing methods from classes as abstract as FilterParameterWidget makes me worried about breaking builds due to expected methods no longer being available, even if the methods themselves probably should not have been available outside of specific instances of this class.

Reply:
I believe this change was actually made in the ongoing feature/OSD_GUI branch being worked on. I don't expect this PR to be ready before that branch is submitted so I put a PR to merge into the develop because I believe the changes in the OSD_GUI branch will be in develop before this guy gets merged in, which should ease the merging a bit.